.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @confluentinc/builds-automations-and-repositories
```

Comment:
Could you update the ownership information in https://github.com/confluentinc/cortex/blob/master/.cortex/catalog/python-github-backup.yaml as well?